### PR TITLE
Create pull request template

### PR DESCRIPTION
**Pull request by Bondcoder**
null

-----
[View rendered .github/pull_request_template.md](https://github.com/CodeYourFuture/cyf-forms/blob/NOTAG-create-template/.github/pull_request_template.md)

### CODE REVIEW

**Comment by Bondcoder** [comment]
There wasn't a template for this repo.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+### Author checklist
+
+<!-- All PRs -->
+
+- [ ] I have written a title that reflects the relevant ticket
+- [ ] I have written a description that says what the PR does and how to validate it
+- [ ] I have linked to the project board ticket (and any related PRs/issues) in the Links section
+- [ ] I have added a link to this PR to the ticket
+- [ ] I have made the PR to `staging` from a branch named `<category>/<name>`, e.g. `feature/edit-spaceships` or `bugfix/restore-oxygen`
+- [ ] I have completed the manual tests [described here](https://github.com/CodeYourFuture/tech-team/wiki/Manual-Test-Procedures)
```
Comment (textbook) [inline .github/pull_request_template.md:36]: This link is broken and I can't see that page in the wiki - @kfklein15 you've moved some things around, do you know where it got to?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+### Author checklist
+
+<!-- All PRs -->
+
+- [ ] I have written a title that reflects the relevant ticket
+- [ ] I have written a description that says what the PR does and how to validate it
+- [ ] I have linked to the project board ticket (and any related PRs/issues) in the Links section
+- [ ] I have added a link to this PR to the ticket
+- [ ] I have made the PR to `staging` from a branch named `<category>/<name>`, e.g. `feature/edit-spaceships` or `bugfix/restore-oxygen`
```
Comment (textbook) [inline .github/pull_request_template.md:35]: This is outdated, we should mention `qa` here too

**Review by textbook — CHANGES_REQUESTED** [review]
I just tried out the template on #99, looks good. Two minor things to review.

File: .github/pull_request_template.md (hunk 1 above)
Comment (SaadiaELF) [reply .github/pull_request_template.md:36]: @kfklein15 , I couldn't find a link to `manual tests` in Wiki page

Comment (textbook) [reply .github/pull_request_template.md:36]: Seems to have moved to https://github.com/CodeYourFuture/tech-team/wiki/3.-How-we-work#manual-test-procedures

**Review by SaadiaELF — APPROVED** [review]

**Review by kfklein15 — APPROVED** [review]
LGTM

**Review by textbook — APPROVED** [review]

**Pull request merged**